Samples/Docker-Simple/Client/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using GrainInterfaces;
+using Orleans;
+using Orleans.Hosting;
+using Orleans.Runtime.Configuration;
+using System;
+using System.IO;
+using System.Threading;
+
+namespace Client
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var connectionString = File.ReadAllText("connection-string.txt");
+
+            var config = new ClientConfiguration
+            {
+                GatewayProvider = ClientConfiguration.GatewayProviderType.AzureTable,
+                DataConnectionString = connectionString
+            };
+
+            var client = new ClientBuilder()
+                .AddApplicationPartsFromBasePath()
+                .UseConfiguration(config)
+                .Build();
+
+            client.Connect().Wait();
```

Comment:
just move all of this to an async method and have Main just call the async method by doing:
```
ExecuteAsync().GetAwaiter().GetResult();
```
Then don't use `.Wait()` nor `.Result` to block. This way exceptions and so on are displayed correctly